build-info-extractor/src/main/java/org/jfrog/build/extractor/scan/DependenciesTree.java

```diff
@@ -0,0 +1,117 @@
+package org.jfrog.build.extractor.scan;
+
+import javax.swing.tree.DefaultMutableTreeNode;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Vector;
+
+/**
+ * @author yahavi
+ */
+public class DependenciesTree extends DefaultMutableTreeNode {
```

Comment:
I suggest adding a description for class, explaining what it is for. This could help the random code reader underatand the context.